Packages/Store/Sources/Requests/ContactRequest.swift

```diff
@@ -0,0 +1,37 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import GRDBQuery
+import GRDB
+import Primitives
+import Combine
+
+public struct ContactListRequest: ValueObservationQueryable {
+    public static var defaultValue: [Contact] { [] }
+    
+    private let searchQuery: String?
+    
+    public init(
+        searchQuery: String?
```

Comment:
```suggestion
        searchQuery: String?
```

I don't think it's used, remove it